lib/cooked_processor_mixin.rb

```diff
@@ -251,6 +253,31 @@ def add_broken_image_placeholder!(img)
     true
   end
 
+  def add_blocked_hotlinked_image_placeholder!(el)
+    el.name = "a"
+    el.set_attribute("href", el[PrettyText::BLOCKED_HOTLINKED_SRC_ATTR])
+    el.set_attribute("class", "blocked-hotlinked-placeholder")
+    el.set_attribute("title", I18n.t("post.image_placeholder.blocked_hotlinked_title"))
+    el << "<svg class=\"fa d-icon d-icon-link svg-icon\" aria-hidden=\"true\"><use href=\"#link\"></use></svg>"
```

Comment:
Sorry I remembered why I said https://github.com/discourse/discourse/pull/16940#discussion_r885378393 and is because of this line. If I make a breaking change to this line here, I don't think our current spec would catch it. Personally, I prefer to have a high level of coverage but not the end of the world if this isn't tested.